code/controllers/subsystem/air.dm

```diff
@@ -244,8 +249,18 @@ var/datum/subsystem/air/SSair
 		if(T.excited_group)
 			T.excited_group.garbage_collect()
 
+/datum/subsystem/air/proc/activate_queued_turfs()
+	if(LAZYLEN(to_activate))
+		for(var/T in to_activate)
+			add_to_active(T, to_activate[T])
+	to_activate = null
```

Comment:
also, the bug is that turfs should not be checking for `atmos_adjacent_turfs` before ssair init (because it's done in ssair init) and that non-open turfs are ending up in the `atmos_adjacent_turfs` list.